frontend/src/components/ContactView.vue

```diff
@@ -198,7 +203,7 @@ export default {
         formattedBirthday() {
             if (!this.contact || !this.contact.birthday) return '';
             const [year, month, day] = this.contact.birthday.split('-');
-            return `${day}.${month}${year && year !== '0001' ? '.' + year : ''}`;
+            return `${day}.${month}.${year && year !== '0001' ? year : ''}`;
```

Comment:
This could be returned as a datetype. Would make it easier for fomatting later on.